### DISCUSSION TITLE
Add "Sandbox Access" Group function to creating Sandboxes

### DISCUSSION THREAD

**Discussion by goto-dev-null** [forcedotcom/cli#3158] [Ideas]
From [docs](https://help.salesforce.com/s/articleView?id=sf.data_sandbox_selective_access.htm&type=5):

> When you create or refresh a Developer and Developer Pro sandbox, you must grant access to the sandbox using a public group.

Functionality pictured below, the new text field at the bottom. You give the *name* of the public Group. This was released in [Summer '24](https://help.salesforce.com/s/articleView?id=release-notes.rn_dev_environments_sandbox_selective_access_more_groups.htm&release=250&type=5).

<img width="717" alt="image" src="https://github.com/user-attachments/assets/d0fbd83a-443e-433a-b4d3-114c2e0994f1" />

It kinda goes without saying what the benefits are and how helpful it would be. Any chance we can get this on the CLI to match with UI?

**Comment by jshackell-sfdc** [comment]
Hi @goto-dev-null. 

Back in October we added two new options to the sandbox definition file: `activationUserGroupId` and `activationUserGroupName`.  See the [release notes](https://github.com/forcedotcom/cli/blob/main/releasenotes/README.md#2618-october-9-2024) and the [reference docs on the sandbox def file](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_sandbox_definition.htm).  Is this what you're requesting?

**Reply by goto-dev-null** [reply]
Yes!! That is exactly it! Should have thought to check the definition file docs. Thanks!
